### PR TITLE
[feat] Cards and Text Utils aka typography

### PR DESCRIPTION
**Pull request by shivampandey0**
## Text Utils
- Headings
- Paragraphs
- Text Styles
- Text Alignments

image

## Cards
- Badge on card (top-right & top-left)
- Dismiss Button on card (top-right & top-left)
- Card with full-overlay
- Text only card
- Horizontal & Vertical Cards
- Cards shadow

image

### CODE REVIEW

**Review by Jia-A — COMMENTED** [review]
Looks good to me. Nice use of comments throughout the code!

File: css/components/cards.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  width: var(--size-4);
+  height: var(--size-4);
+  border-radius: var(--radius-round);
+
+  opacity: 0.6;
+  cursor: pointer;
+}
+
+.card-icons .icon {
+  font-size: var(--size-0);
```
Comment (Hamdrive) [inline css/components/cards.css:46]: Why have a size 0?

File: css/helpers/flexbox.css [hunk 2]
```diff
@@ -17,7 +17,18 @@
...
+
+/* Justify */
+.justify-sb{
+  justify-content: space-between;
+}
+
+
 /* flex gaps */
+.gap-05 {
+  gap: var(--size-000);
```
Comment (Hamdrive) [inline css/helpers/flexbox.css:29]: using 0 like this looks confusing
if you are referring to miniscule values, use names instead

**Review by FarhanMobashir — COMMENTED** [review]
all good

**Review by Hamdrive — COMMENTED** [review]
Good work done, suggestion given for variable names with 0's

File: css/components/cards.css (hunk 1 above)
Comment (shivampandey0) [reply css/components/cards.css:46]: It's not size 0 but 1rem.

It maybe needs to be more thought out in the future but right now I needed these naming to better remember and not go back again and again to declarations to check, that's why.

File: css/helpers/flexbox.css (hunk 2 above)
Comment (shivampandey0) [reply css/helpers/flexbox.css:29]: The answer is given above, I may change this naming paradigm in the future or add a declaration about this in docs. Right now it makes me remember the variables easily.

**Pull request merged**